cypress/e2e/spec.cy.js

```diff
@@ -0,0 +1,12 @@
+describe('template spec', () => {
+  it('passes', () => {
+    // go to the Skosmos front page
+    cy.visit('http://localhost/Skosmos')
+    // click on the first vocabulary in the list
+    cy.get('#vocabulary-list').find('a').first().click()
+    // check that we are still within the Skosmos URL space
+    cy.url().should('include', 'http://localhost/Skosmos/')
```

Comment:
This could fail is others have already port `80` used and somehow tell Cypress to use another port number. But easy to fix later if needed :+1:

Reply:
Yes, this needs to be more general, also in order to run the same tests under GitHub Actions.

Reply:
Left for later.